.github/workflows/update_galata_references.yaml

```diff
@@ -16,9 +16,9 @@ jobs:
   update-snapshots:
     if: >
       (
-        github.event.issue.author_association == 'OWNER' ||
-        github.event.issue.author_association == 'COLLABORATOR' ||
-        github.event.issue.author_association == 'MEMBER'
+        github.event.comment.author_association == 'OWNER' ||
+        github.event.comment.author_association == 'COLLABORATOR' ||
+        github.event.comment.author_association == 'MEMBER'
```

Comment:
Can we maybe add `github.event.issue.author_association == 'CONTRIBUTOR'` so that it can be able to run for authors who've previously contributed?

Reply:
Great! I was also hoping for that behavior but didn't know how to do it. If you'd like to open a PR to get credit for this change that'd be great. Otherwise I can do it.

Reply:
Thanks for validating, I'll be more than happy to add it :) 
This is quite a non-trivial change so I can also do it in #96 if that's fine, If not I can quickly open a separate follow-up PR.

Reply:
Both are fine thanks!

Reply:
Since #96 is merged already, I've opened #98 for this.